azure-local/concepts/compare-windows-server.md

```diff
@@ -69,60 +69,65 @@ The following table compares the workloads and benefits of Azure Local and Windo
+| Azure Kubernetes Service (AKS)| Yes | Yes, but sunset <sup>1</sup> |
 | Azure Arc-Enabled PaaS Services | Yes | Yes |
-| Windows Server 2022 Azure Edition | Yes | No|
+| Windows Server 2022 Azure Edition | Yes | No |
+| Windows Server 2025 Azure Edition | Yes | No |
+| Windows Server Management Features via Arc (Nov. 2024) | Yes | Yes |
 | Windows Server subscription add-on (Dec. 2021)| Yes | No |
-| Free Extended Security Updates (ESUs) for Windows Server and SQL 2008/R2 and 2012/R2 | Yes | No <sup>1</sup>|
+| Free Extended Security Updates (ESUs) for Windows Server and SQL 2008/R2 and 2012/R2 | Yes | No <sup>2</sup> |
 
- <sup>1</sup> Requires purchasing an Extended Security Updates (ESU) license key and manually applying it to every VM.
+ <sup>1</sup> [AKS on Windows Server End of Support 27th March 2027](https://learn.microsoft.com/en-us/azure/aks/aksarc/aks-windows-server-retirement?tabs=ws).<br>
+ <sup>2</sup> Requires purchasing an Extended Security Updates (ESU) license key and manually applying it to every VM.
  
 ## Compare technical features
 
-The following table compares the technical features of Azure Local and Windows Server 2022.
-
-| Attribute | Azure Local | Windows Server 2022 |
-| ------------- | ------------------- | ----------------------- |
-| Hyper-V | Yes | Yes |
-| Storage Spaces Direct | Yes | Yes |
-| Software-Defined Networking | Yes | Yes |
-| Adjustable storage repair speed | Yes | Yes|
-| Secured-core Server| Yes | Yes |
-| Stronger, faster network encryption | Yes | Yes |
-| 4-5x faster Storage Spaces repairs | Yes | Yes |
-| Stretch clustering for disaster recovery with Storage Spaces Direct<sup>1</sup> | Yes | No |
-| High availability for GPU workload | Yes | No |
-| Restart up to 10x faster with kernel-only restarts | Yes | No |
-| Simplified host networking with Network ATC | Yes | No |
-| Storage Spaces Direct on a single machine | Yes | No |
-| Storage Spaces Direct thin provisioning | Yes | No |
-| Dynamic processor compatibility mode| Yes | No |
-| Cluster-Aware OS feature update | Yes | No |
-| Integrated driver and firmware updates | Yes (Integrated Systems only) | No |
-
-<sup>1</sup> Functionality only available in Azure Local, version 22H2.
+The following table compares the technical features of Azure Local and Windows Server 2025.
+
+| Attribute | Azure Local 23H2 / 24H2 | Windows Server 2025 | Windows Server 2022
+| ------------- | ------------------- | ----------------------- | ----------------------- |
+| Hyper-V | Yes | Yes | Yes |
+| Storage Spaces Direct | Yes | Yes | Yes |
+| Software-Defined Networking | Yes | Yes | Yes |
+| Adjustable storage repair speed | Yes | Yes | Yes |
+| Secured-core Server| Yes | Yes | Yes |
+| Stronger, faster network encryption | Yes | Yes | Yes |
+| 4-5x faster Storage Spaces repairs (256kb slab size) | Yes | Yes | Yes |
+| Stretch clustering for disaster recovery with Storage Spaces Direct | No <sup>1</sup> | Work in Progress | No |
```

Comment:
the previous version gave readers the impression that stretched clustering is still supported with Azure Local 23H2 and later. I have corrected that in a best possible way adding a sup note.